lib/search/service.rb

```diff
@@ -17,11 +17,11 @@ class Service < Common::Client::Base
     configuration Search::Configuration
 
     attr_reader :query
-    attr_reader :offset
+    attr_reader :page_number
 
-    def initialize(query, offset = 0)
+    def initialize(query, page_number = 1)
```

Comment:
In keeping with the following Kam's schema suggestions for precedence, those endpoints use `page` instead of `page_number` for the endpoint query param.  

https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/prescriptions/indexPrescriptions

IMO, we should stay consistent and use `page` here, as well.  Si?